### ISSUE TITLE
Idea- Weird Idea-- Multiple mods at once

### ISSUE THREAD

**Issue by realzguy** [DarkSnakeGang/GoogleSnakeCustomMenuStuff#123]
could you make a way to have multiple mods at once so i could play, for say, more pudding and mouse mod both at once? i dont know, just a thought

**Comment by hwatson381** [comment]
Yeah, we're planning to do this, but it's a bit complicated as some mods are incompatible with others.

ps. It's better to leave issues here instead https://github.com/DarkSnakeGang/GoogleSnakeModLoader/issues

**Comment by Yarmiplay** [comment]
This has been added with Chimera Mod
